docs/pages/docs/quint.md

```diff
@@ -229,7 +229,7 @@ Options:
                                                                         [string]
   --out-itf      output the trace in the Informal Trace Format to file, e.g.,
                  out_{seq}.itf.json where {seq} is the trace sequence number
-                 (suppresses all console output)                        [string]
+                                                                        [string]
```

Comment:
You updated the documentation, but can you also update the code for this?
https://github.com/informalsystems/quint/blob/main/quint/src/cli.ts#L241